opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json

```diff
@@ -167,17 +222,121 @@
                   "chk_difficult_breathing",
                   "chk_unable_to_suck",
                   "chk_umbilical_discharge",
-                  "chk_no_movement",
                   "chk_convulsions",
                   "chk_skin_rashes",
                   "chk_pale",
-                  "chk_eye_discharge",
                   "chk_severe_abdominal_pain",
-                  "chk_lethargy"
+                  "chk_lethargy",
+                  "chk_urination"
                 ]
               }
             ]
           }
+        },
+        "v_required": {
+          "value": "true",
+          "err": "Please select one"
+        }
+      },
+      {
+        "key": "discuss_neonatal_advice_health_facility",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "",
+        "type": "toaster_notes",
+        "text": "Discuss with the mother about the advice she received for these conditions when she sent the Baby to the Health facility including Counselling, treatment when to follow-up.",
+        "toaster_type": "info",
+        "relevance": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "pnc_danger_signs_baby_relevance.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "issue_referral_neonatal",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "",
+        "type": "toaster_notes",
+        "text": "You should send the Baby to the health facility as soon as possible and within 24 hours for further screening.",
+        "text_color": "#CF0800",
+        "toaster_type": "problem",
+        "relevance": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "pnc_danger_signs_baby_relevance.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "neonatal_referral_health_facility",
+        "type": "native_radio",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "114403AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```

Comment:
We are phasing out the use of openmrs uuids here, please add the same value as the key